cogs/remind_me.py

```diff
@@ -139,12 +138,17 @@ async def autocomplete_get_delays(ctx: TeXBotAutocompleteContext) -> set[str]:
 
                 slice_size: int
                 for slice_size in range(1, len(formatted_time_choice) + 1):
-                    if match.group("ctx_time_choice").casefold() == formatted_time_choice[:slice_size]:  # noqa: E501
+                    if (
+                        match.group("ctx_time_choice").casefold()
+                        == formatted_time_choice[:slice_size]
+                    ):
```

Comment:
See above comment about moving boolean expressions to separate flag rather than splitting lines